apps/demo/src/app/cart/components/cart-wrapper-summary/cart-wrapper-summary.component.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, Input, Directive } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { StoreModule, combineReducers, MemoizedSelector, Store } from '@ngrx/store';
+
+import { DaffCartFactory } from '@daffodil/cart/testing';
+import { fromCart, DaffCart } from '@daffodil/cart';
+
+import { CartWrapperSummaryComponent } from './cart-wrapper-summary.component';
+import * as cartSelector from '../../selectors/cart-selector';
+import { provideMockStore, MockStore } from '@ngrx/store/testing';
+import { cold } from 'jasmine-marbles';
+
+@Component({ template: '<demo-cart-wrapper-summary [cart]="cartValue"></demo-cart-wrapper-summary>' })
+class WrapperComponent {
+  cartValue: DaffCart;
+}
+
+@Component({
+  selector: 'demo-cart-totals',
+  template: ''
+})
+class MockCartTotalsComponent {
+  @Input() cart: DaffCart;
+}
+
+@Directive({
+  selector: '[demoProceedToCheckout]'
+})
+class MockProceedToCheckoutDirective { }
+
+describe('CartWrapperSummary', () => {
+  let wrapper: WrapperComponent;
+  let fixture: ComponentFixture<WrapperComponent>;
+  let cartWrapperSummaryComponent: CartWrapperSummaryComponent;
```

Comment:
You want the component being tested to be named `component`? Should I make this change in the other cart components as well?

Reply:
This is something that we've been doing as come across it in our tests. It would be too tedious to do this across all files at the same time. Additionally, it would make reviews too long. For now, I would just address it in the new components that you've made here.